standards/cryptography.md

```diff
@@ -0,0 +1,57 @@
+---
+expires: 2019-03-15
+---
+# Cryptography
+
+## The base principle
+
+All data **must** employ adequate and proportionate cryptography to preserve confidentiality and integrity whether data is at-rest or in-transit.
```

Comment:
Second base principle: do not attempt to come up with your own cryptographic algorithms
Third principle: do not write your own implementation of cryptographic protocols
Fourth: use well-known implementations at the highest possible abstraction level.

Reply:
PR inbound